IOIOLibAndroid/build.gradle

```diff
@@ -1,16 +1,5 @@
 apply plugin: 'com.android.library'
-
-uploadArchives {
-    repositories {
-        mavenDeployer {
-            pom.project {
-                name = 'IOIOLibAndroid'
-                description = 'IOIO library for Android - common code with ADB support'
-                packaging = 'aar'
-            }
-        }
-    }
-}
+apply plugin: 'com.github.dcendents.android-maven'
```

Comment:
I'm confused about this. It looks like this project [was abandoned](https://github.com/dcendents/android-maven-gradle-plugin). What is this for?

Reply:
To make jitpack work

Reply:
Sorry, yes. I had forgotten [this](https://jitpack.io/docs/ANDROID/). Thought this was something new for some reason.